.gitignore

```diff
@@ -11,6 +11,9 @@
 /db/*.sqlite3
 /db/*.sqlite3-journal
 
+# Ignore Rubymine files
+/.idea
```

Comment:
editors are a personal choice so i encourage people to add these to their [personal gitignores](https://stackoverflow.com/questions/7335420/global-git-ignore) instead of adding to every project

Reply:
@TomWoodward yup. thanks.  good tip.  

i added this to my ~/.gitconfig, and then inside this file, added my editor's files to exclude.  now i never have to add rubymine to gitignore. i'm stoked! 

[core]
	excludesfile = ~/.local_gitignore